packages/sdk/src/verbs.ts

```diff
@@ -41,10 +46,27 @@ export class Verbs {
   }
 
   /**
-   * Get the lend provider instance
-   * @returns LendProvider instance if configured, undefined otherwise
+   * Get lend operations interface
+   * @description Access to lending operations like markets and vault information.
+   * Throws an error if no lend provider is configured in VerbsConfig.
+   * @returns LendReadOperations interface for lending operations
+   * @throws Error if lend provider not configured
+   */
+  get lend(): LendReadOperations {
+    if (!this._lend) {
+      throw new Error(
+        'Lend provider not configured. Please add lend configuration to VerbsConfig.',
+      )
+    }
+    return this._lend
+  }
```

Comment:
I could see us eventually abstracting this to be true for all optional namespaces